lib/tasks/coinmarketcap.rake

```diff
@@ -2,10 +2,10 @@ require 'httparty'
 
 namespace :coinmarketcap do
   namespace :cronjobs do
-    # Direct ticker access
-    desc "10min ticker price update"
+    # Direct suppy access
+    desc "Daily suppy update"
     task :ticker_update => :environment do
```

Comment:
I'm reverting back to the original code since this is for the current ticker update. New stuff doesn't touch this file, this is still just the northpole rake task.